workspace-loader/tsconfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "include": [
+    "src"
+  ],
+  "compilerOptions": {
+    /* Basic Options */
```

Comment:
JSON is not allowing comments, so all comments should be removed
you may check on any online JSON validator

Reply:
Done